internal/booking/chalmers/bookingService.go

```diff
@@ -99,7 +99,7 @@ func (bs BookingService) MyBookings() ([]booking.Booking, error) {
 	bookings := make([]booking.Booking, 0, 4)
 	selectedDate := ""
 	for _, tr := range trs {
-		headline := tr.Find(".headline.changeDateLink")
+		headline := tr.Find(".headline.leftRounded.t")
```

Comment:
Would it be possible to find a css selector that seems more stabel?

Reply:
There don't seem to be given the current approach to the problem. Maybe we can re-write this part if we figure out a better, more reliable, way of doing it